src/core/error_handling.c

```diff
@@ -225,6 +225,12 @@ char* error__js_filename_string = "???.js";
 EM_JS_NUM(errcode, error_handling_init_js, (), {
   Module.handle_js_error = function(e)
   {
+    if (e instanceof Module._PropagatePythonError) {
+      // Python error indicator is already set in this case. If this branch is
+      // not taken, Python error indicator should be unset, and we have to set
+      // it. I think in this case we don't want to tamper with the traceback?
```

Comment:
```suggestion
      // it. In this case we don't want to tamper with the traceback.
```
otherwise it's not clear what the reader is supposed to do with questions in comments.